src/technologies/overview/tools-libraries/index.md

```diff
@@ -1,5 +1,61 @@
 ---
-title:
+title: Tools and libraries
 ---
 
-{% include content-not-available.md %}
\ No newline at end of file
+To work with the utilities provided by the Magento PWA project, you should be familiar with the tools and libraries described on this page.
+
+## Webpack
+
+[Webpack] is a configuration-driven module bundler for JavaScript applications.
+
+Webpack's primary function is to create one or more bundles from the dependencies in your project's modules.
+Webpack is also able to transform, package, or optimize resources and assets using plugins.
+This allows you to extend webpack's functionality beyond JavaScript bundling.
```

Comment:
Should "webpack" be "Webpack" (name of a product/piece of software/specific thing)?

Reply:
`webpack` team is very persistent about this one, actually. https://webpack.js.org/branding/

> webpack should always be written in lower-case letters, even at the beginning of a sentence.